src/System.Linq.Expressions/src/System/Runtime/CompilerServices/DebugInfoGenerator.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.Emit;
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Generates debug information for lambdas in an expression tree.
+    /// </summary>
+    public abstract class DebugInfoGenerator
+    {
+        /// <summary>
+        /// Creates PDB symbol generator.
+        /// </summary>
+        /// <returns>PDB symbol generator.</returns>
+        public static DebugInfoGenerator CreatePdbGenerator()
+        {
+            // Creating PDBs is not suported in .NET Core
+            throw new PlatformNotSupportedException();
```

Comment:
Is this not supported yet, not supported unless it gets screamed for and/or implemented by the interested or never to be supported?